lib/kubernetes-deploy/runner.rb

```diff
@@ -112,6 +112,21 @@ def template_variables
 
     private
 
+    # Inspect the file that was mentioned in the kubectl output
+    # to make it easier for developer to understand what's going on
+    def inspect_kubectl_out_for_yaml(out)
+      # Output example:
+      # error: unable to decode "/tmp/path/to/file": [pos 96]: json: expect char '"' but got char '{'
+      if match = out.match(/error:(.+)"(?<path>\S+)":+/)
```

Comment:
This looks pretty prone to inspecting the wrong file.. is there a directory somewhere of what the errors will look like? Could we have a regex that'll look for things that look more like files?